src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,77 @@
+package duke;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Storage {
+    /**
+     * Used to load tasks from a storage file.
```

Comment:
Method header comments should start in the form Returns ..., Sends ..., Adds ...